packages/mediacenter/kodi/scripts/cputemp

```diff
@@ -24,51 +24,50 @@
 TEMP=0
 
 if [ $(basename "$0") = "gputemp" -o "$1" = "gpu" ]; then
-  if which lspci >/dev/null; then
+
+  if which lspci 2>/dev/null; then
     if lspci -n | grep 0300 | grep -q 10de; then
       [ -x /usr/bin/nvidia-smi ] && TEMP=`/usr/bin/nvidia-smi -q -x | grep '<gpu_temp>' | awk '{ print $1 }' | sed 's,<gpu_temp>,,g'`
```

Comment:
`grep 0300 | grep -q 10de` is also a bit suspect.

We want to match:
```
01:00.0 0300: 10de:0a64 (rev a2)
```
but could match:
```
01:00.0 10de: beef:0300 (rev a2)
```
or some other variation.

The following is a bit "safer":
```
  if [ -x /usr/bin/nvidia-smi ]; then
    if command -v lspci >/dev/null && lspci -n | grep -q "0300: 10de:"; then
      TEMP="$(/usr/bin/nvidia-smi -q -x | awk '/<gpu_temp>/ { print $1 }' | sed 's,<gpu_temp>,,g')"
    fi
  fi
```